.github/workflows/renovate.yml

```diff
@@ -0,0 +1,57 @@
+name: Renovate
```

Comment:
wait, was Renovate not running on crossplane-runtime since switching to self hosted?

Reply:
I think it was still using the Renovate app. See for example https://github.com/crossplane/crossplane-runtime/pull/695 which was opened last week.

I'm guessing we should switch to self hosted for the same reasons as c/c though.